databroker/src/broker.rs

```diff
@@ -165,6 +196,8 @@ pub struct NotificationError {}
 
 #[derive(Debug, Clone, Default)]
 pub struct EntryUpdate {
+    pub id: Option<i32>,
```

Comment:
In my opinion EntryUpdate should have one of `ìd` or `path` other one could be get by some functions. Does not make sense to save both.